travis/python_linter.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
-# usage: 
+# usage:
 #   * set LINT_PYTHON to be the arguments to flake8 (e.g., package name)
 #   * source `python_linter.sh` on Travis CI
 
+cat <<EOF 1>&2
+WARNING: This script has been deprecated.
+Please use the new shared configuration yamls instead.
+Instructions for use can be found in the README.md at:
+https://github.com/pcdshub/pcds-ci-helpers
+EOF
+
 pip install flake8
 
 LINT_PYTHON_PATH=${LINT_PYTHON_PATH:-$TRAVIS_BUILD_DIR}
```

Comment:
The use of this variable seems a little redundant as `PYTHON_LINT` is also meant to take on directory information. Can I change this to simply cd to the root directory?

Reply:
Seems fine. Or do you think someone would find it confusing to have `PYTHON_LINT` take a path?

Reply:
The README seemed to describe that as the intended usage and the examples gave it paths.

Reply:
I think the project is young enough that we can try to come up with good patterns without sticking to any names we chose on a whim while setting up the repo. 

I don't feel strongly about this case - but maybe give it some thought.

Reply:
I've changed `LINT_PYTHON` to `PYTHON_LINT_OPTIONS` for now. Is that kinda what you had in mind? I don't think the extra length hurts at all.

Reply:
Yeah, looks good.

Reply:
This of course introduced an API change that I should've more clear about. `LINT_PYTHON` is being used in `typhos`'s CI config and possibly others. It's an easy fix but I should've noticed that would happen beforehand.